link_release.sh

```diff
@@ -9,20 +9,20 @@ sha_macos=""
 sha_windows=""
 if test -f "${DL_LINUX}";
 then 
-  echo hello
   sha_linux=shasum -a 256 "$DL_LINUX"
+  echo sha_linux
 fi
 
 if test -f "${DL_MACOS}";
 then 
-  echo hello
   sha_macos=shasum -a 256 "$DL_MACOS"
+  echo sha_macos
 fi
 
 if test -f "${DL_WIN}";
 then 
-  echo hello
   sha_windows=shasum -a 256 "$DL_WIN"
+  echo sha_windows
```

Comment:
just echo these as debugging,
think to clean this up we can use an array 

maybe something like:
```sh
DL_FILES=($DL_LINUX $DL_MACOS $DL_WIN)
for i in "${DL_FILES[@]}"
do
    declare "sha_$i=$(shasum -a 256 $i)"
done
```

we can probably pass the args without preventing file not found with directly
```sh
--arg sha_linux "$(sha256sum $DIR_LINUX)"
```
or something like that to clean up